### DISCUSSION TITLE
Image size increase so much.

### DISCUSSION THREAD

**Discussion by Madtoyit** [ltsp/ltsp#299] [Q&A]
My installation is on ubuntu 20.04 with default DE Gnome.
Usually i do an image rebuilt after system update about 3/4 weeks.

I noticed that the size of the image x86_64.img file in /srv/ltsp/images increases a lot every time.
from May to August I went from 3,5GB to 5GB.

-rw-r--r-- 1 root root 3412373504 mag 29 07:41 x86_64.img
-rw-r--r-- 1 root root 3947401216 giu  7 08:08 x86_64.img
-rw-r--r-- 1 root root 3983392768 giu 13 07:32 x86_64.img
-rw-r--r-- 1 root root 4064411648 giu 21 18:18 x86_64.img
-rw-r--r-- 1 root root 4380123136 lug 11 18:53 x86_64.img
-rw-r--r-- 1 root root 4438450176 lug 25 19:43 x86_64.img
-rw-r--r-- 1 root root 4971634688 ago 15 11:21 x86_64.img

At this rate I will soon find a huge img file.
Obviously no new program was installed, only security and periodic updates.

What am I wrong?

Maybe the problem are the updates of some applications that are installed via snap? I see that they have passed in the img?
or what?

```
user02@nbmiolenovo129:~$ df -hl
File system          Dim. Usati Dispon. Uso% Montato su
udev                 3,8G     0    3,8G   0% /dev
tmpfs                768M  3,6M    765M   1% /run
/run/initramfs/ltsp  3,8G   17M    3,8G   1% /
tmpfs                3,8G     0    3,8G   0% /dev/shm
tmpfs                5,0M  4,0K    5,0M   1% /run/lock
tmpfs                3,8G     0    3,8G   0% /sys/fs/cgroup
/dev/loop1            97M   97M       0 100% /snap/core/9665
/dev/loop3            56M   56M       0 100% /snap/core18/1885
/dev/loop2            97M   97M       0 100% /snap/core/9804
/dev/loop4           161M  161M       0 100% /snap/gnome-3-28-1804/116
/dev/loop5            55M   55M       0 100% /snap/core18/1880
/dev/loop6           251M  251M       0 100% /snap/gimp/292
/dev/loop8           261M  261M       0 100% /snap/kde-frameworks-5-core18/32
/dev/loop7           256K  256K       0 100% /snap/gtk2-common-themes/9
/dev/loop9           162M  162M       0 100% /snap/gnome-3-28-1804/128
/dev/loop11          256M  256M       0 100% /snap/gnome-3-34-1804/36
/dev/loop10           63M   63M       0 100% /snap/gtk-common-themes/1506
/dev/loop12          256M  256M       0 100% /snap/gnome-3-34-1804/33
/dev/loop15           65M   65M       0 100% /snap/stickynotes/950
/dev/loop13           30M   30M       0 100% /snap/snapd/8790
/dev/loop14           50M   50M       0 100% /snap/snap-store/467
/dev/loop16          256K  256K       0 100% /snap/gtk2-common-themes/13
/dev/loop17          177M  177M       0 100% /snap/gimp/281
/dev/loop18           44M   44M       0 100% /snap/snap-store/415
/dev/loop19           30M   30M       0 100% /snap/snapd/8542
tmpfs                768M   16K    768M   1% /run/user/125
tmpfs                768M   24K    768M   1% /run/user/1002
```

tks.

**Comment by alkisg** [comment]
One method to compare sizes is:

```shell
sudo mount -o loop,ro /srv/ltsp/images/x86_64.img /mnt
sudo du -sh /mnt/*
```

You do this for two different images (x86_64.img and x86_64.old), and you see for example that the problem is in /mnt/lib. Then you can go a level deeper with `sudo du -sh /mnt/lib/*`, and that way you locate the folder that got bigger.
